src/main/java/htsjdk/samtools/reference/ReferenceSequenceFileFactory.java

```diff
@@ -81,7 +81,7 @@ public static ReferenceSequenceFile getReferenceSequenceFile(final File file, fi
      * @param preferIndexed if true attempt to return an indexed reader that supports non-linear traversal, else return the non-indexed reader
      */
     public static ReferenceSequenceFile getReferenceSequenceFile(final File file, final boolean truncateNamesAtWhitespace, final boolean preferIndexed) {
-        return getReferenceSequenceFile(file.toPath(), truncateNamesAtWhitespace, preferIndexed);
+        return getReferenceSequenceFile(IOUtil.toPath(file), truncateNamesAtWhitespace, preferIndexed);
```

Comment:
Like the changes above, the changes in this file won't change the code behavior as an NPE will still be generated, just later in the call chain.